### PR TITLE
Ease seeing appraisal while inputting it - move window

### PR DESCRIPTION
**Pull request by Eccenux**
Not sure if there is some refactoring going on, but here is my take on "Ease seeing appraisal while inputting it #373".

Tried detecting swipe gestures, but that's a mess... So I just move the info window up/down when apprise is open.

To save some space I changed first option for appraisal spinners to serve as a label.

Also shortened phrases for Valor.

### CODE REVIEW

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java [hunk 1]
```diff
@@ -741,7 +751,23 @@ public void toggleMoreResultsBox() {
...
+     */
+    private void moveInfoWindow(boolean toTop) {
+        if (toTop) {
+            layoutParams.gravity = Gravity.CENTER | Gravity.TOP;
+            Log.d(TAG, "moveInfoWindow: top");
+        } else {
+            layoutParams.gravity = Gravity.CENTER | Gravity.BOTTOM;
+            Log.d(TAG, "moveInfoWindow: bottom");
+        }
+        windowManager.updateViewLayout(infoLayout, layoutParams);
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: I'd expect this doesn't go all the way to the top? That's not enough—on a Nexus 5 one needs to go all the way to the top *and* to hide other widgets for this to work.

**Comment by Blaisorblade** [comment]
Hi, thanks for your PR! But there's already #382 with a similar approach, maybe you can compare? In particular see:
https://github.com/farkam135/GoIV/pull/382/commits/a40d4d0352fc4a732abc9f3eed25bcbc5a965250

Also another PR, #379 by @imapokemon4, makes the phrases more complete, not shorter. I have no strong opinion on this.

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java (hunk 1 above)
Comment (Eccenux) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: There is a lot of space once labels are gone. Below is on Xperia Z5 Compact (smaller then Nexus 5).
goiv_appraisal_

Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: > There is a lot of space once labels are gone.

This... actually looks good. Is there a need to shorten phrases? I'll try merging the rest.

Comment (Eccenux) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: > This... actually looks good. Is there a need to shorten phrases? I'll try merging the rest.

Short phrases, no, not really. But you would need to preserve the first option in each group. This need to be something meaningful (as there are no labels).

Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: What do you think of 7f3fdcad402cbc9ab92787a70ed7a24baf8a6f52? That might need further refinement. @DJCrapsody @nahojjjen and @imapokemon4 what do you think? (I'd probably at least go for capitals).

> But you would need to preserve the first option in each group. This need to be something meaningful (as there are no labels).

I've kept your first label there. Having something meaningful could suggest longer phrases.

**Comment by Blaisorblade** [comment]
Closing this—let's continue the discussion on labels in #379.

**Pull request closed without merging**